<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>681</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-radius: 20px;
border: 5px dotted rgb(87, 63, 136);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 141, 103);
border-radius: 20px;
font: 11pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>DETECT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 141, 103);
border-radius: 20px;
font: 11pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>UNDETECT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 141, 103);
border-radius: 20px;
font: 11pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>PREDICT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>100</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 141, 103);
border-radius: 20px;
font: 11pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>UNPREDICT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>411</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px dotted black;
border-radius: 50px;
font:150pt &quot;Mongolian Baiti&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>19</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SHOWING CAMERA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuADD">
    <property name="title">
     <string>ADD DATA</string>
    </property>
    <addaction name="menuAdd"/>
   </widget>
   <addaction name="menuADD"/>
  </widget>
  <action name="menuAdd">
   <property name="text">
    <string>ADD NEW DATA</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
